src/stratis_cli/_error_reporting.py

```diff
@@ -56,6 +56,15 @@ class StratisCliErrorCodes(IntEnum):
 )
 
 
+def exit_(code, msg):
```

Comment:
Why ```exit_```, the underscore after the name? Is there a convention that this is following?